resources/measures/upgrade_hvac_economizer/tests/hvac_economizer_test.rb

```diff
@@ -171,11 +169,156 @@ def apply_measure_and_run(test_name, measure, argument_map, osm_path, epw_path,
     return result
   end
 
+  def get_design_oa_flow_rates(model)
+    hash_oa_design_rates = {}
+    # get OA design rates prior to measure implementation
+    model.getSizingSystems.each do |sizing_system|
+
+      # get related airloophvac
+      air_loop_hvac = sizing_system.airLoopHVAC
+      name_air_loop_hvac = air_loop_hvac.name.to_s
+
+      # get design OA flow rate
+      oa_design_rate = nil
+      if sizing_system.autosizedDesignOutdoorAirFlowRate.is_initialized
```

Comment:
This is not a reliable field for getting the design outdoor air flow rate. This value matches the supply air flow rate to size the system as if it is 100% OA (worst case scenario). It is debatable if this is realistic, but that's the current state.

![image](https://github.com/NREL/ComStock/assets/50876267/f0a700f7-d418-4b2c-97a3-275f0b686824)

The "Minimum Outdoor Air Flow Rate" field for the "Controller:OutdoorAir" class is more reliable, but if using this, you need to check that it is non zero, which happens for some DCV modeling.

Reply:
added a new test `test_requested_oa_rates`